### ISSUE TITLE
Not all buttons  are Buttons and they don't indicate disability outside color in Admin.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#788]
**Describe the bug**
Buttons don't indicate disability outside color in Design. They are displayed on screen but are not functional.

**To Reproduce**
1. Go to Admin (http://localhost:9992/cm/app/?view=workflow&site=test3&path=%2FSites%2Ftest3%2F&pathType=page)
2. Double Click on Site.
3. The delete button against the workflow is not a button and don't indicate disability outside color.
4. Found same under the Roles and Users tab.

**Expected behavior**
Disability should be indicated and it should be buttons.
